scripts/split-e2e-tests-v2.ts

```diff
@@ -14,6 +14,7 @@ const RUN_SOLO = [
   'src/__tests__/geo-remove-3.test.ts',
   'src/__tests__/geo-update-1.test.ts',
   'src/__tests__/geo-update-2.test.ts',
+  'src/__tests__/hostingPROD.test.ts',
```

Comment:
this test keeps derailing other tests in retry loop. isolating.
https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/16824/workflows/9ab1103b-4e53-421b-8766-f89571eb2dd9/jobs/817847

![image](https://user-images.githubusercontent.com/5849952/222608000-dc9ad769-8fa5-4073-b630-b61d8ab04df5.png)